contents/docs/web-analytics/web-vitals.mdx

```diff
@@ -0,0 +1,148 @@
+
+At the bottom of the dashboard, you can see a table with a breakdown of the pages with good, needs improvement, and poor scores. You should focus on optimizing the pages with poor scores.
+
+
+### What are the thresholds for good Web Vitals?
+
+Web Vitals can be categorized in 3 categories: **Good**, **Needs improvement**, and **Poor**.
+
+They differ according to the specific metric:
+
+| Metric                          | Good    | Needs improvement | Poor    |
+| ------------------------------- | ------- | ----------------- | ------- |
+| FCP (First Contentful Paint)    | ≤ 1.8s  | 1.8s - 3.0s       | > 3.0s  |
+| LCP (Largest Contentful Paint)  | ≤ 2.5s  | 2.5s - 4.0s       | > 4.0s  |
+| INP (Interaction to Next Paint) | ≤ 200ms | 200ms - 500ms     | > 500ms |
+| CLS (Cumulative Layout Shift)   | ≤ 0.1   | 0.1 - 0.25        | > 0.25  |
+
+### What is the recommended percentile?
+
+PostHog allows you to look at performance for p75, p90 and p99. We suggest you to use the p90 percentile to verify your web vitals. This means that 90% of your users will have an experience better than the one displayed. This is for a couple reasons:
+
+- It is the most common percentile used by web developers to verify their web vitals.
+- It provides a good balance between capturing real user issues while filtering out extreme outliers that may be due to factors outside your control (like very poor network conditions or device performance)
+- Looking at p99 may lead to unnecessary over-optimization. Web Vitals are a good starting point for improving your site, but they are not the only metrics you should focus on.
+
+
+## How to enable web vitals autocapture
+
+Go to your [Autocapture settings](https://app.posthog.com/settings/project-autocapture#web-vitals-autocapture). Then, in the **Web vitals autocapture** section, click **Enable**.
+
+> **Note:** Web vitals autocapture is separate from our regular [autocapture](/docs/product-analytics/autocapture) feature. Web vitals can be captured regardless of whether autocapture is enabled.
+
+> **Note:** You'll need to be using SDK v1.141.2+ to use web vitals autocapture.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/v1726058265/posthog.com/contents/Screenshot_2024-09-11_at_1.36.27_PM.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/v1726058265/posthog.com/contents/Screenshot_2024-09-11_at_1.36.34_PM.png"
+  alt="How to enable web vitals autocapture" 
+  classes="rounded"
+/>
+
+Once enabled, web vitals are captured under the `$web_vitals` event name. These events include properties for each of the metrics like `$web_vitals_FCP_value` and `$web_vitals_FCP_event`. 
+
+You can choose which metrics to capture by using the toggles.
+
+
+### Configuring web vitals autocapture
+
+You can provide a client side config to tune web vitals autocapture. These are set on [the `capture_performance` key](https://github.com/PostHog/posthog-js/blob/main/src/types.ts#L209).
```

Comment:
It looks like the line number changed for the `capture_performance` key. I wonder if we should maybe just point to the file without the line number.

```suggestion
You can provide a client side config to tune web vitals autocapture. These are set on [the `capture_performance` key](https://github.com/PostHog/posthog-js/blob/main/src/types.ts) in the `PostHogConfig` interface.
```